src/lib/routes/admin-api/tag.ts

```diff
@@ -74,7 +73,7 @@ class TagController extends Controller {
             method: 'post',
             path: '',
             handler: this.createTag,
-            permission: UPDATE_FEATURE,
+            permission: NONE,
```

Comment:
Removed project level permission to create tag, because it is not related to project/feature